test/tools/merkleTree.js

```diff
@@ -0,0 +1,135 @@
+const { keccak256, bufferToHex } = require('ethereumjs-util');
+
+class MerkleTree {
+    constructor (elements) {
+        // Filter empty strings and hash elements
+        this.elements = elements.filter(el => el).map(el => keccak256(el));
```

Comment:
Why not 
```suggestion
        this.elements = elements.map(el => keccak256(el));
```
Is the filter condition not always true?

Reply:
first you filter out the undefined, 0 elements or you would have keccak256(0) in the map

Reply:
in js false, undefined, 0, ''null, are the same thing in bool if you not not use ===. So when if(value) I test that is not one of those